### PR TITLE
[Feature][UI Next][V1.0.0-Alpha] Adjust all table column widths in th…

### LINKED ISSUE #9040: [Feature][UI Next][V1.0.0-Alpha] Adjust all table column widths in this project.
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

Uniform all the tables style in this project as the task instance table.

### Use case

image

### Related issues

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by Amy0104**
## Purpose of the pull request

This pull request adjusts all table column widths in this project.
## Releated issues

close #9040

### CODE REVIEW

**Comment by songjianet** [comment]
image

File: dolphinscheduler-ui-next/src/utils/column-config.ts [hunk 1]
```diff
@@ -0,0 +1,94 @@
+/*
```
Comment (songjianet) [inline dolphinscheduler-ui-next/src/utils/column-config.ts]: This file should be update to be column-width-config.ts

**Review by songjianet — CHANGES_REQUESTED** [review]

File: dolphinscheduler-ui-next/src/utils/column-config.ts (hunk 1 above)
Comment (Amy0104) [reply dolphinscheduler-ui-next/src/utils/column-config.ts]: Got it,I will change the file name to column-width-config and the COLUMN_CONFIG to COLUMN_WIDTH_CONFIG.

**Review by songjianet — APPROVED** [review]
LGTM

**Pull request merged**